backup-calendars.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+URL="$1"
+ 
+mkdir -p /tmp/calendar
+cd /tmp/calendar
+#fetch filelist:
+cadaver $URL << EOC | grep .ics | sed 's/.ics.*/.ics/' | sed 's/\ *//' > ./filelist
```

Comment:
This command fails on my development environment (probably because cadaver is not installed there). 

Without installing it, I can't check if the sync works. Is there a way that makes this available via composer or npm? I'd rather not rely on dependencies that one has to remember to install.

Reply:
this is not php or nodejs, this is a unix command line tool available with the package manager of you distro
also you have to configure the authentification
i have linked the man page in the ticket

Reply:
Is there no way around this? I'd rather not have a dependency that has to be installed manually, all dependencies should be installed via npm / composer (at least in my opinion). 

Maybe we can use some other tool that can be installed via one of those channels?

Reply:
none of the php dependencys i found worked anymore, i don't know why, thats why i start to think in  a diffrent way

Reply:
If this is only used for the BD-Sync, I think thats ok. But generally, I'd rather not have dependencies that outside of a package-manager (also, this make the BD-Sync not work on Windows machines).